app/services/requirements/file_attachment_upload_checker.rb

```diff
@@ -28,6 +28,8 @@ class FileAttachmentUploadChecker
       "text/xml", # xml, xsd
     ].freeze
 
+    ALLOWED_ZIP = "application/zip" # zip
```

Comment:
I found this quite confusing so I had Ben explain to me about how that zip is a little special. 

I think there's probably a way to make this easier to understand.

I'd suggest we include this mime_type in the list of allowed ones and then just check that it's not an archive in the method.